styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java

```diff
@@ -83,7 +83,16 @@ public void printEvents(List<EventInfo> eventInfos) {
 
   @Override
   public void printBackfill(Backfill backfill) {
-    System.out.println(backfill);
+    System.out.println(String.format("id: %s", backfill.id()));
```

Comment:
Actually we should have a single backfill info in one line, otherwise the output is really confusing